src/app/components/articleList/articleList.tsx

```diff
@@ -18,11 +19,12 @@ export default function ArticleList() {
         </Typography>
       </div>
       <CardList />
-      <div>
+      <div className={styles.buttonArea}>
         <ActionButton
           label='Explore more'
           disabled={false}
```

Comment:
Leave the `disabled` all together. Set the Button component to accept disabled={false} by default. Same thing is applicable for `fullWidth`.